game/src/ecs/components/AnimationComponent.java

```diff
@@ -57,7 +65,12 @@ public AnimationComponent(@DSLContextMember(name = "entity") Entity entity) {
     }
 
     /**
-     * @param animation new current animation of the entity
+     * Set the current animation displayed on entity. The animation passed does not have to be one
+     * of the animations stored in this component. It can be any animation. If the animation passed
+     * is not displayed on the entity, there may be another point in the code where the animation is
```

Comment:
"point in code" => overwritten by another system?

Reply:
Naja es muss ja nicht zwingend ein System sein, welches die Animation überschreibt. Wobei das ECS es *normalerweise* so vorsieht, ja.